app/services/streaming_service_provider_data_service.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module StreamingServiceProviderDataService
+  class << self
+    def get_providers(tmdb_id:, title:, media_type:, media_format:, release_date: nil)
+      params = { tmdb_id: tmdb_id, media_type: media_type }
+      results = Tmdb::Client.request(:streaming_service_providers, params)&.dig(:results, :US)
+
+      parameterized_title = title.parameterize(separator: '+')
```

Comment:
This evening when I was annoyed with this [`ActiveSupport::Inflector.parameterize` method](https://apidock.com/rails/ActiveSupport/Inflector/parameterize), I did a little more digging and found the [`String` class's version](https://apidock.com/rails/v5.2.3/String/parameterize) which actually does allow me to pass a separator. 😄

Reply:
nice! 😎